controllers/admin/AdminLegacyLayoutController.php

```diff
@@ -93,7 +93,7 @@ public function initContent()
             'page_header_toolbar_btn' => $this->page_header_toolbar_btn,
         );
 
-        if (!empty($this->helpLink)) {
+        if ($this->helpLink === false || !empty($this->helpLink)) {
```

Comment:
Easier way to be able to disable Help link, just set `'help_link' => null` by default just above :)

Reply:
And it prevents templates change and mix between null & false everywhere

Reply:
There is already a behavior when $this->helpLink is null => overriden in [AdminController](https://github.com/PrestaShop/PrestaShop/blob/develop/classes/controller/AdminController.php#L1577).
We have 3 use cases:
- help_link is defined in a controller => this value is used
- help_link is null or empty => auto generated value from Parent controller
- help_link is false => disabled

Reply:
So ... what do we do here? :angel:

Reply:
I was waiting for a rage quit :( 
But since you're still here, I must validate your PR :heart: